tests/llmcompressor/transformers/compression/recipes/new_quant_fp8.yaml

```diff
@@ -0,0 +1,19 @@
+quant_stage:
+  quant_modifiers:
+    GPTQModifier:
```

Comment:
Can we do this with just quantization modifier?

We shouldn't need GPTQ for this case

Reply:
Can do as follow up

Reply:
Gotcha, I just updated the PR with this change (both in the example and the integration test). However the nice scheme/target UX is only for GPTQModifier so it made the example a bit messier. I'll move the scheme parsing to QM in a follow up PR